packages/studio-plugin/tests/__fixtures__/ParsingOrchestrator/src/components/Card.tsx

```diff
@@ -1,3 +1,4 @@
+import "../index.css";
```

Comment:
can we test a nested CSS import as well to make sure the dependencies are being traced correctly?

Reply:
done!